jena-arq/src/main/java/org/apache/jena/riot/system/Checker.java

```diff
@@ -21,44 +21,38 @@
 
 import org.apache.jena.graph.Node ;
 import org.apache.jena.graph.Triple ;
-import org.apache.jena.iri.IRI ;
 import org.apache.jena.riot.checker.* ;
+import org.apache.jena.sparql.core.Quad;
 
-/** A checker validates RDF terms. */
+/** A checker that drives the process of validating RDF terms, triples and quads. */
 public final class Checker
 {
     private boolean allowRelativeIRIs = false ;
     private boolean warningsAreErrors = false ;
     private ErrorHandler handler ;
-    
+
     private NodeChecker checkLiterals ;
     private NodeChecker checkURIs ;
     private NodeChecker checkBlankNodes ;
     private NodeChecker checkVars ;
 
-    public Checker()
-    {
-        this(null) ;
+    public Checker() {
+        this(null);
     }
-    
-    public Checker(ErrorHandler handler)
-    {
+
+    public Checker(ErrorHandler handler) {
         if ( handler == null )
-            handler = ErrorHandlerFactory.getDefaultErrorHandler() ;
-        this.handler = handler ;
-        
-        checkLiterals = new CheckerLiterals(handler) ;
-       
-        checkURIs = new CheckerIRI(handler, IRIResolver.iriFactory()) ;
-        checkBlankNodes = new CheckerBlankNodes(handler) ;
-        checkVars = new CheckerVar(handler) ;        
-    }
-
-    public ErrorHandler getHandler()                { return handler ; } 
-    public void setHandler(ErrorHandler handler)    { this.handler = handler ; }
-    
-    public boolean check(Node node, long line, long col)
-    {
+            handler = ErrorHandlerFactory.getDefaultErrorHandler();
```

Comment:
Extra spaces?

Reply:
The previous line is `if ( handler == null )` -- the diff doesn't make it very clear, it's decided that's an unchanged line as seval changes happen in the same area.

Reply:
Will fix directly along with some clearing up.